core/src/com/unciv/logic/map/MapUnit.kt

```diff
@@ -847,6 +849,13 @@ class MapUnit {
             .sumBy { it.params[0].toInt() }
     }
 
+    fun receivedInterceptDamageFactor(): Float {
+        return getMatchingUniques("Damage taken from interception reduced by []%")
+            .fold(1f) { product, it -> product * 1f - it.params[0].toFloat() / 100f }
```

Comment:
I believe you missed some brackets here.
I personally prefer doing this with a for loop instead of `.fold()`, but that is a matter of opinion.

Reply:
Please do this with a for loop, as it's more immediately readable.

Reply:
It's too bad Kotlin has no `productBy`. I have replaced this with a for loop.